ios/FluentUI/HUD/HUDTokens.swift

```diff
@@ -10,13 +10,13 @@ import UIKit
 open class HeadsUpDisplayTokens: ControlTokens {
 
     /// The color of the squared background of the Heads-up display.
-    open var backgroundColor: DynamicColor { aliasTokens.colors[.backgroundInverted] }
+    open var backgroundColor: DynamicColor { aliasTokens.colors[.backgroundDarkStatic] }
 
     /// The color of the activity indicator presented by the Heads-up display.
-    open var activityIndicatorColor: DynamicColor { aliasTokens.colors[.foreground2] }
+    open var activityIndicatorColor: UIColor { .init(light: UIColor(dynamicColor: aliasTokens.colors[.foregroundLightStatic]).withAlphaComponent(0.4), dark: UIColor(dynamicColor: aliasTokens.colors[.foregroundLightStatic]).withAlphaComponent(0.6)) }
```

Comment:
With #1258 merged, I think adding a token of type `UIColor` is a bit of a problem. I think we may be better off keeping this returning `DynamicColor`, and putting the different opacities somewhere else. Though, I also don't really get why we need the different opacities.

Reply:
The different opacities were set by design to get the desired contrast ratios. Can you clarify why we shouldn't return a `UIColor`?

Reply:
With `ControlTokenValue`, each token is actually a collection of every possible type, so adding a new type would actually lead to a lot of bloat that we don't want.